hack/verify-structured-logging.sh

```diff
@@ -46,9 +50,35 @@ done < <(echo "${all_packages[@]}" | tr " " "\n")
 
 ret=0
 echo -e "\nRunning structured logging static check on all packages"
-GOOS=linux    logcheck "${packages[@]}" || ret=$?
-GOOS=windows  logcheck "${packages[@]}" || ret=$?
+
+# Function to run logcheck and capture output
+run_logcheck() {
+  local os="$1"
+  shift
+  local output
+  output=$(GOOS="$os" logcheck "$@" 2>&1)
+  echo "$output"
+  if echo "$output" | grep -q 'unstructured logging function'; then
+    echo "Error: Unstructured logging detected. Please fix the issues."
+    return 1
+  fi
+}
+
+# Run logcheck for Linux
+echo "Running logcheck for Linux..."
+if ! run_logcheck linux "${packages[@]}"; then
+  ret=1
+fi
+
+# Run logcheck for Windows
+echo "Running logcheck for Windows..."
+if ! run_logcheck windows "${packages[@]}"; then
+  ret=1
+fi
```

Comment:
I think previous code to run against **both** linux and windows is a fault. We can remove running against windows.

Reply:
done